tests/redirects/map_external.py

```diff
@@ -9,5 +9,6 @@
 URLS = flatten((
     url_test('https://www.mozilla.com/', 'https://www.mozilla.org/firefox/'),
     url_test('https://aurora.mozilla.org/', 'https://www.mozilla.org/firefox/aurora/', status_code=requests.codes.found),
-    url_test('https://beta.mozilla.org/', 'https://www.mozilla.org/en-US/firefox/channel/#beta'),
+    # Bug 1299947 once new /channel pages are live we can update this redirect test.
+    # url_test('https://beta.mozilla.org/', 'https://www.mozilla.org/en-US/firefox/channel/#beta'),
```

Comment:
I've decided it might be better to leave this redirect test in, as it may well still pass (the other above it aren't final destinations either). If it does fail for whatever reason, we can always update it then.